.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: ci
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    env:
+      COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache node modules
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('**/package.json') }}
```

Comment:
I realize the root package does not have one, but would it be better to also include the `package-lock.json` files? That way the cache invalidates on changes to transient dependencies, not just the deps in the `package.json`. :thinking:

Reply:
I agree, but since I don't know why the root package does not have one and since it's [intentionally excluded](https://github.com/nearform/graphql-hooks/blob/master/.npmrc), I didn't want to modify it as the reason why it's excluded may be valid but we may not know it.

Reply:
Funny though, because tracing back to that change where lockfile was removed, it seems there was an overall intention to remove all package lock files, even from packages. But packages do have them now, so apparently that intention was lost over time. That's one reason why I'm cautious reverting existing decisions because we may not know why they were made.

Reply:
Strange.

In the end, it's probably not wise to make a change as I suggested in this PR. Probably best to stick with the original functionality, just as you've done.

Reply:
@codyzu can you approve the PR then please?